src/auth/auth.js

```diff
@@ -46,7 +47,7 @@ export default function authentication(config) {
   });
 
   app.use((req, res, next) => {
-    if (req.isAuthenticated()) {
+    if (req.isAuthenticated() && !isExpired(req.session.passport.user)) {
```

Comment:
Maybe we should also also remove the cookie  whenever it is expired/invalid to avoid leaving dead "sessions" laying around in browser cookies?

ie something like... 

```
...
if (isExpired(token)) { 
res.headers[`Set-Cookie`] = 'token=deleted; path=/; expires=Thu, 01 Jan 1970 00:00:00 GMT';
res.redirect(req.url)
}
...
```

Reply:
Could we also set the cookie expiry date to the same as what we have in the jwt token so the browser drops it anyway?